frontend/frontend-react/src/components/template/Main.css

```diff
@@ -39,15 +41,22 @@ div.content {
 }
 
 .image {
-    width: 400px;
     height: 400px;
 
     border: 2px solid black;
 }
 
 .buttons {
-    width: 400px;
+    width: 100%;
     height: 30px;
+    display: flex;
+    justify-content: space-between;
+}
 
-    border: 2px solid black;
+ul {
+    list-style: none;
+}
+
+.fa-paper-plane {
+    font-size: 25px;
 }
```

Comment:
'No new line at end of file' alert

Reply:
You can use [EditorConfig](https://editorconfig.org/) for this in vscode and many others code editors, just make sure to enable the `insert_final_newline` option, vscode will handle it automatically for you if you have the `Format On Save` option enabled on workspace settings. You can also take a look on [Prettier](https://prettier.io/) which is more advanced, but can format your code automatically.